### PR TITLE
Bug 1730190 - filter out overactive clients in events_daily queries

### PR DESCRIPTION
**Pull request by akkomar**
This fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1730190.
Cutoff threshold of 3M filters out two clients that had unusually high number of submissions.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

### CODE REVIEW

File: bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql
Comment (wlach) [inline bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql:22]: I feel like it would be good to add this same filtering/counting for glean apps, where we might encounter the same problem. Can you add this?

File: bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql
Comment (wlach) [inline bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql:45]:
```suggestion
    -- filter out overactive clients: they distort the data and can cause the job to fail: https://bugzilla.mozilla.org/show_bug.cgi?id=1730190
```

**Review by wlach — CHANGES_REQUESTED** [review]
Thanks @akkomar! I have one requested change but overall LGTM

Also, feel free to add me as a co-owner for these queries. I'm honestly not super with familiar with them yet, but I will be continuing to work in this space.

File: bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql
Comment (akkomar) [reply bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql:22]: I'd opt for not adding this to glean query at this point. Distribution of event counts per client in this table does not have such long tail like in telemetry. I don't really know what cutoff level would make sense in the long term.

**Comment by akkomar** [comment]
> Thanks @akkomar! I have one requested change but overall LGTM
> 
> Also, feel free to add me as a co-owner for these queries. I'm honestly not super with familiar with them yet, but I will be continuing to work in this space.

Great! Added you as an owner.

See my comment inline about glean.

File: bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql
Comment (wlach) [reply bigquery_etl/events_daily/query_templates/events_daily_v1/query.sql:22]: I feel like it makes sense to have some kind of sanity check here, just to guard against something weird happening unexpectedly in the future. We can deliberately set the threshold high for now (3,000,000 seems like a fine number to me). That said, I don't feel strongly enough about this to block merging this as-is.

**Review by wlach — APPROVED** [review]

**Pull request merged**
